nss-day-cohort-43/nuttyshell-plenty-of-witches
# Pk event data provider

# Description
Allows the user to create and see events. User will need to login to see the events container
## Type of change
Please delete options that are not relevant.
- [x] Able to see events
- [x] Able to add new events
- [x] Able to delete events that user creates (can't delete other users events.)
- [x] The user must login in to see the events elements
# Testing Instructions
Make sure that you have Nutshell.js called in main.js The loadEvents function should be call in that module. That should bring you to the main page were you will be required to log in before you can make events. (All of this should be in your code when you pull this branch.)
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
